setup.py

```diff
@@ -1,24 +1,28 @@
 #!/usr/bin/env python
 
 import os
+
 from setuptools import setup
 
-setup(name='zict',
-      version='2.0.0',
-      description='Mutable mapping tools',
-      url='http://zict.readthedocs.io/en/latest/',
-      maintainer='Matthew Rocklin',
-      maintainer_email='mrocklin@gmail.com',
-      license='BSD',
-      keywords='mutable mapping,dict,dask',
-      packages=['zict'],
-      install_requires=open('requirements.txt').read().strip().split('\n'),
-      long_description=(open('README.rst').read() if os.path.exists('README.rst')
-                        else ''),
-      classifiers=[
-          "Programming Language :: Python",
-          "Programming Language :: Python :: 3.6",
-          "Programming Language :: Python :: 3.7",
-          "Programming Language :: Python :: 3.8",
-      ],
-      zip_safe=False)
+setup(
+    name="zict",
+    version="2.0.1",  # Fix this before mergin. Used to test distributed
```

Comment:
```suggestion
    version="2.0.0.dev1",
```
PEP-440 compliant version. Tested:
```python
>>> LooseVersion("2.0.0.dev1") > "2.0.0"
True
```